pre_commit_hooks/detect_aws_credentials.py

```diff
@@ -0,0 +1,65 @@
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import argparse
+import ConfigParser
```

Comment:
py2/py3 issue.  Feel free to add `six` as a dependency and `from six.moves import configparser`